### ISSUE TITLE
OS.Unmapper takes unused parameter (owner) in its constructor

### ISSUE THREAD

**Issue by minborg** [OpenHFT/Chronicle-Core#112]
It might look like this:

```
    public static class Unmapper implements Runnable {
        private final long size;
        private final ReferenceCounted owner;

        private volatile long address;

        public Unmapper(long address, long size, ReferenceCounted owner) throws IllegalStateException {

            assert (address != 0);
            this.address = address;
            this.size = size;
            this.owner = owner;
        }

        @Override
        public void run() {
            if (address == 0)
                return;

            try {
                unmap(address, size);
                owner.release();
                address = 0;

            } catch (@NotNull IOException e) {
                Jvm.warn().on(OS.class, "Error on unmap and release", e);
            }
        }
    }
```

**Comment by minborg** [comment]
Can you review the proposed code above @RobAustin ?

**Comment by minborg** [comment]
Changing like above will create exceptions in queue cleaners

**Comment by peter-lawrey** [comment]
This has been fixed

**Comment by hft-team-city** [comment]
Released in [Chronicle-Core-2.20.42](https://github.com/OpenHFT/Chronicle-Core/releases/chronicle-core-2.20.42), [BOM-2.20.67](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.67)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Core-2.20.102](https://github.com/OpenHFT/Chronicle-Core/releases/chronicle-core-2.20.102), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
